tab_ecosystem.md

```diff
@@ -27,6 +27,12 @@ and is online-readable **for free** at
 edition is also available **for free** at
 <https://leanpub.com/juice-shop> in PDF, Kindle and ePub format.
 
+## MultiJuicer Platform
```

Comment:
Maybe something more like?
So that it's clear its an official part of juice shop?
Currently its between two other headlines starting with official, making it seem like this might not be official.

```suggestion
## Official Multi User Platform: MultiJuicer
```

Reply:
> Maybe something more like?
> So that it's clear its an official part of juice shop?
> Currently its between two other headlines starting with official, making it seem like this might not be official.
> 
> ```suggestion
> ## Official Multi User Platform: MultiJuicer
> ```

Good idea, then we can even go with just "Official Multi User Platform" and have the name just in the logo.